### PR TITLE
build: automate and document release process

### PR DESCRIPTION
**Pull request by lucab**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,22 +41,16 @@ SRC_DIRS := cli pkg
...
-    BASEIMAGE?=armel/busybox
-endif
-ifeq ($(ARCH),arm64)
-    BASEIMAGE?=aarch64/busybox
-endif
-ifeq ($(ARCH),ppc64le)
-    BASEIMAGE?=ppc64le/busybox
+        IMARCH?=""
+else
+        IMARCH?='-$(ARCH)'
```
Comment (squeed) [inline Makefile]: whoops. I made a mistake in the tectonic-torcx makefile; these single quotes are incorrect.

Comment (lucab) [reply Makefile]: Ack, fixed to use double-quotes instead.

**Comment by lucab** [comment]
@squeed just realized there was this stale PR pending, thus rebased. PTAL.

**Comment by lucab** [comment]
(Regarding `ppc64le`, we currently do not have that on ContainerLinux radar, so not worrying about that at the moment)

**Pull request merged**